api/v1alpha1/accesslogging_types.go

```diff
@@ -19,6 +19,10 @@ type ProxyAccessLog struct {
 type ProxyAccessLogSetting struct {
 	// Format defines the format of accesslog.
 	Format ProxyAccessLogFormat `json:"format"`
+	// Matches defines the match conditions for accesslog in CEL expression.
+	// Invalid [CEL](https://www.envoyproxy.io/docs/envoy/latest/xds/type/v3/cel.proto.html#common-expression-language-cel-proto) expressions will be ignored.
```

Comment:
can we link to a page that shows the CEL examples ? along with all available attributes

Reply:
it's on that page
<img width="909" alt="image" src="https://github.com/envoyproxy/gateway/assets/4354057/7852283e-c391-4b20-b426-d322ed9651ae">